specification/api/components/requests/LetterUpdateRequest.yml

```diff
@@ -0,0 +1,12 @@
+required: true
```

Comment:
May be worth matching the filename to the corresponding endpoint `patchLetterRequest.yml`